automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/PingSpec.scala

```diff
@@ -1,10 +1,10 @@
 package org.broadinstitute.dsde.workbench.leonardo
 
 import org.broadinstitute.dsde.workbench.auth.UserAuthToken
-import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
-import org.scalatest.{FreeSpec, Matchers}
+import org.scalatest.{DoNotDiscover, FreeSpec, Matchers}
 
-class PingSpec extends FreeSpec with Matchers with LeonardoTestUtils with BillingFixtures {
+@DoNotDiscover
```

Comment:
Since the tests is discovered as part of `LeonardoTestSuite`, we put `@DoNotDiscover` here so it does not run twice. Unfortunately you need to comment out `@DoNotDiscover` if you want to run the test individually via sbt.